bigquery/google/cloud/bigquery/client.py

```diff
@@ -122,11 +122,13 @@ class Client(ClientWithProject):
     """The scopes required for authenticating as a BigQuery consumer."""
 
     def __init__(
-            self, project=None, credentials=None, _http=None, location=None):
+            self, project=None, credentials=None, _http=None,
+            location=None, query_job_config=None):
```

Comment:
ISTM this argument should be named `default_query_job_config`.  It needs to be added to the class docstring above, too.

Reply:
Tests need to accommodate this renaming, too.